samcli/commands/build/command.py

```diff
@@ -418,3 +438,34 @@ def _process_env_var(container_env_var: Optional[List[str]]) -> Dict:
             processed_env_vars[location_key][env_var_name] = value
 
     return processed_env_vars
+
+
+def _process_image_options(image_args: Optional[Tuple[str]]) -> Dict:
+    """
+    Parameters
+    ----------
+    image_args : Tuple
+        Tuple of command line image options in the format of
+        "Function1=public.ecr.aws/abc/abc:latest" or
+        "public.ecr.aws/abc/abc:latest"
+
+    Returns
+    -------
+    dictionary
+        Function as key and the corresponding image URI as value.
+        Global default image URI is contained in the None key.
+    """
+    build_images: Dict[Union[str, None], str] = dict()
+    if image_args:
+        for build_image_string in image_args:
+            function_name: Union[str, None] = None
+            if "=" in build_image_string:
+                parts = build_image_string.split("=", 1)
+                function_name = parts[0]
```

Comment:
Are we adding any checks here so that function_name and image_uri won't be empty?

Reply:
Added empty checks